### PR TITLE
scraper endpoint

### PR DESCRIPTION
**Pull request by yingjieMiao**
@raymondng

### CODE REVIEW

File: kifi-backend/modules/common/app/com/keepit/scraper/ScraperServiceClient.scala [hunk 1]
```diff
@@ -212,6 +214,13 @@ class ScraperServiceClientImpl @Inject() (
       Json.fromJson[Option[ExtendedEmbedlyInfo]](r.json).get
     }
   }
+
+  def getURISummaryFromEmbedly(uri: NormalizedURI, minSize: ImageSize, descriptionOnly: Boolean): Future[Option[URISummary]] = {
+    val payload = Json.obj("uri" -> uri, "minSize" -> minSize, "descriptionOnly" -> descriptionOnly)
```
Comment (raymondng) [inline kifi-backend/modules/common/app/com/keepit/scraper/ScraperServiceClient.scala:219]: minor: do you want to consider using a more strongly typed payload?

**Comment by raymondng** [comment]
lgtm

File: kifi-backend/modules/common/app/com/keepit/scraper/ScraperServiceClient.scala (hunk 1 above)
Comment (yingjieMiao) [inline kifi-backend/modules/common/app/com/keepit/scraper/ScraperServiceClient.scala:219]: valid concern. probably can factor out a request class.

Comment (yingjieMiao) [inline kifi-backend/modules/common/app/com/keepit/scraper/ScraperServiceClient.scala:219]: we already have `URISummaryRequest`. it contains a few more fields. 
When we finally move everything to scraper, we can use that as a typed payload.

Comment (raymondng) [inline kifi-backend/modules/common/app/com/keepit/scraper/ScraperServiceClient.scala:219]: Cool

**Pull request merged**